package.json

```diff
@@ -21,6 +21,10 @@
     "semver": "^7.3.8",
     "simple-git": "^3.14.1"
   },
+  "engines": {
+    "node": ">=16.9.1",
+    "npm": ">=7 <8"
```

Comment:
Settings this to `<8` because using `npm@8` would overwrite the current `v2` file with a `v3` one.